docs/durable_execution/prefect.md

```diff
@@ -255,19 +255,21 @@ from prefect import flow
 from pydantic_ai import Agent
 from pydantic_ai.durable_exec.prefect import PrefectAgent
 
-agent = Agent(
-    'openai:gpt-4o',
-    name='daily_report_agent',
-    instructions='Generate a daily summary report.',
-)
 
-prefect_agent = PrefectAgent(agent)
-
-@flow
+@flow(log_prints=True)
 async def daily_report_flow(user_prompt: str):
     """Generate a daily report using the agent."""
+    agent = Agent(
+        'openai:gpt-4o',
+        name='daily_report_agent',
+        instructions='Generate a daily summary report.',
+    )
+
+    prefect_agent = PrefectAgent(agent)
```

Comment:
I guess the old code didn't work because the agent can't be serialized. But agents are typically defined at the top level, often in a separate module, so this is a gotcha the user will need to be aware of. Can we document it explicitly above? And perhaps raise a useful error if they do the obvious-but-wrong thing?